### PR TITLE
Closes Issue #47

### PR DESCRIPTION
**Pull request by m-qin**
Closes issue #47

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,11 +8,24 @@
...
 
-**Analysis**: To run all the analysis and regenerate the results,
+**Analysis**: To run the analyses and regenerate the results,
 
-* Request the `data_input_private.zip` to the authors. Unzip in this location [data/input/private](./data/input/private/).
-* Install [CausalGPS package](https://github.com/cran/CausalGPS) R package
-* Run:
+* Request `data_input_private.zip` from the authors. Unzip in this location [data/input/private](./data/input/private/).
+* Install [CausalGPS package](https://github.com/cran/CausalGPS) R package (use version 0.2.8.9 or higher)
+* Decide whether to run the Generalized Estimating Equations (GEE) models (1 causal analysis and 1 sensitivity analysis). GEE models may take up to 96 GB to run (and up to 250 GB for the "trim 1/99" sensitivity analysis), which is more memory than a typical computer contains.
```
Comment (audiracmichelle) [inline README.md:15]: remove: which is more memory than a typical computer contains

File: code/continuous_treatment_causal_analyses.R [hunk 2]
```diff
@@ -1,3 +1,5 @@
+##### Note: GEE model (line 65) may take more memory to run (up to 96 GB for trim 5/95, or up to 250 GB for trim 1/99) than is available on some computers #####
```
Comment (audiracmichelle) [inline code/continuous_treatment_causal_analyses.R:1]: remove: than is available on some computers

File: code/sensitivity_analyses/gee_associational_model.R [hunk 3]
```diff
@@ -1,4 +1,4 @@
-##### Note: GEE model may take more memory to run than is available on some computers #####
+##### Note: GEE model may take more memory to run (up to 96 GB) than is available on some computers #####
```
Comment (audiracmichelle) [inline code/sensitivity_analyses/gee_associational_model.R:1]: It is ok to remove "available in some computers"

**Review by audiracmichelle — CHANGES_REQUESTED** [review]
I made minimal easy to resolve suggestions

File: lib/functions_using_gps.R [hunk 4]
```diff
@@ -16,7 +16,7 @@ all_matching_results_1model <- function(seed,
                                         data,
                                         trim,
                                         cat_covariate_names,
-                                        run_gee_model = F, # GEE model requires a lot of memory to run, some computers may not be able to
+                                        run_gee_model = F, # GEE model requires a lot of memory to run (48-72 GB), some computers may not be able to
```
[the remark this answers is not in the document]
Comment (m-qin) [reply lib/functions_using_gps.R]: Thanks for pointing that out! I just made a commit removing or shortening those clarification comments. Does it look better now?

**Review by audiracmichelle — APPROVED** [review]

**Pull request merged**